httplib.h

```diff
@@ -130,9 +130,9 @@
 #pragma comment(lib, "ws2_32.lib")
 
 #ifdef _WIN64
-using ssize_t = __int64;
+using ssize_t = long long;
```

Comment:
@oculusdrifts, I would like to leave it as `__int64`, so that it matches the exactly same as what [the Microsoft documentation describes](https://docs.microsoft.com/en-us/windows/win32/winprog/windows-data-types#size_t). Could you try `__int64` on your machine if it works?

<img width="476" alt="image" src="https://user-images.githubusercontent.com/357397/181645604-8e481326-3080-4f83-bc81-7501dc6bbb17.png">

<img width="710" alt="image" src="https://user-images.githubusercontent.com/357397/181645723-8337190a-1c5b-438f-89a6-26792bd72cee.png">